eng/build.ps1

```diff
@@ -174,9 +174,8 @@ try {
   }
 }
 catch {
-  Write-Host $_
-  Write-Host $_.Exception
   Write-Host $_.ScriptStackTrace
+  Write-PipelineTelemetryError -Category "Build" -Message $_
```

Comment:
Is this file is the only relevant change? It looks like you pulled in some other commits from another PR.

Reply:
You're right—I pulled in the parallelism branch by accident. I'll sort that out.